stellarphot/core.py

```diff
@@ -104,14 +104,14 @@ class BaseEnhancedTable(QTable):
     Parameters
     ----------
 
-    table_description: dict or dict-like
+    table_description: dict or dict-like (Default: None)
```

Comment:
OK, I felt like I should not state it is optional since it's only left that way to have QTable behavior inheritance to work right. :/, but yeah, technically true.  Fixing.